go.mod

```diff
@@ -0,0 +1,60 @@
+module github.com/giantswarm/helmclient
+
+go 1.13
+
+require (
+	github.com/DATA-DOG/go-sqlmock v1.4.1 // indirect
+	github.com/Masterminds/goutils v1.1.0 // indirect
+	github.com/Masterminds/semver v1.5.0
+	github.com/Masterminds/sprig v2.22.0+incompatible // indirect
+	github.com/dsnet/compress v0.0.1 // indirect
+	github.com/giantswarm/backoff v0.2.0
+	github.com/giantswarm/errors v0.1.0
```

Comment:
Should be `v0.2.0`. We need to merge https://github.com/giantswarm/errors/pull/33

Reply:
Updated, @wtty-fool PTAL